### PR TITLE
board support: add Toradex SOM board config files

### PR DESCRIPTION
**Pull request by raja-sivasankaran**
Add Toradex SOM products Tegra, iMX6, iMX7 variants board config files

### CODE REVIEW

File: contrib/board_files/apalis-t30.conf [hunk 1]
```diff
@@ -0,0 +1,137 @@
+# Toradex Apalis T30 Computer On Module.
+# http://developer.toradex.com/products/apalis-t30
+
+[board]
+dtfile = /proc/device-tree/model
```
Comment (bhuvanchandra) [inline contrib/board_files/apalis-t30.conf]: @raja-sivasankaran  does this work on Nvidia's L4T kernel ?

**Comment by bhuvanchandra** [comment]
@raja-sivasankaran does the `dtfile = /proc/device-tree/model` work on non-DT Nvidia L4T kernel.

**Comment by jackmitch** [comment]
Apologies I missed this, you need to squash the last commit as that file is added in an earlier one.

As for the L4T kernel it will need a separate configuration file, however as it's not a mainline kernel the custom support for it doesn't belong in libsoc and should be shipped with the vendor distro.

**Comment by raja-sivasankaran** [comment]
@jackmitch, I have done rebase and Could you please merge the pull request.

**Comment by raja-sivasankaran** [comment]
@jackmitch, I have done rebase and could you please check and merge the pull request.

**Comment by jackmitch** [comment]
Hi,

You need to rebase to remove the merge and also remove the commit 59f7510 which was fixing something that was wrong with your previous commits.

Thanks,

**Pull request closed without merging**